parlai/crowdsourcing/tasks/acute_eval/README.md

```diff
@@ -62,6 +64,8 @@ This is a template of the expected format with the minimal expected fields:
       ]
     }
 
+Note that we assume that "dialogue" consists of strictly alternating turns (e.g. speakers a, b, a, b, a...). Speakers that we would like to evaluate in the dialogues of `dialogue_dicts` should appear in the same order as `speakers_to_eval`; consequently, the number of entries should be the same between the values of these fields. See `task_config/pairings.jsonl` for examples of the format required.
```

Comment:
Hmm - could you clarify "the number of entries should be the same between the values of these fields"? i.e. each turn should have the same number of speakers?

Reply:
Hmm, how about "the number of dialogues in `dialogues_dicts` should be the same as the number of speakers in `speakers_to_eval`? 

Was trying to figure out a way to make 

> Additionally, `speakers_to_eval` must be in the same order as the dialogue_dicts.

a bit more clear relative to

```
                eval_speakers = [
                    s
                    for d in convo_pair["dialogue_dicts"]
                    for s in d["speakers"]
                    if s in convo_pair["speakers_to_eval"]
                ]
                # make sure order is preserved
                assert eval_speakers == convo_pair["speakers_to_eval"]
              
```

though not sure if the wording I settled on was much better, lol.

Reply:
Yeah, I think that's good! Or maybe even "Consequently, each speaker listed in `speakers_to_eval` will correspond to a different dialogue in `dialogue_dicts`", whichever you prefer